app_feup/lib/controller/local_storage/app_exams_database.dart

```diff
@@ -64,4 +86,14 @@ class AppExamsDatabase extends AppDatabase {
 
     await db.delete('exams');
   }
+
+  static FutureOr<void> migrate(
+      Database db, int oldVersion, int newVersion) async {
+    final batch = db.batch();
+    if (oldVersion< newVersion){
+      batch.execute('DROP TABLE IF EXISTS exams');
+      batch.execute(createScript);
+    }
+    await batch.commit();
+  }
```

Comment:
Do we need to migrate the database? It would seem the internal representation hasn't changed

Reply:
Yes, I don't think it will be necessary. I removed the migration code in the new push.